resources/xip-patch/templates/clusterrole.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
When thing about it it probably shouldn't be `ClusterRole`, but two separate `Role`s. One of them should allow access to services in `istio-system` and second to config maps in `kyma-installer`. Current scope is too broad.